### DISCUSSION TITLE
Help how to refresh configuration v1 state?

### DISCUSSION THREAD

**Discussion by Marlo461** [hassio-addons/addon-zwavejs2mqtt#427] [General]
Hello everyone, I need some advice. I use Fibaro FGT-001 heads for heating with radiators. They work well, but I want to use the function (heating medium demand reports) that this head has and read its status (provide heat in order to maintain set temperature). With this state, I want to control the gas boiler. I encountered a problem that is probably caused by the fact that this head is a Z-Wave FLiRS. The problem is that the header changes the status of the request, but the status is sometimes sent and sometimes not. I tested it for a few days and found out that even when the thermostat is OFF, this request is sometimes active, but when the Z-Wave JS UI on this device was manually restored (configuration v1), this status is loaded correctly. My question is whether I know how to create an automation that would perform status recovery (configuration v1). Please advise how I can request a renewal using automation or a script.
I am also attaching a photo before and after the refresh.
[Before refresh configuration v1](https://files.fm/f/mjk76csra)
[After refresh configuration v1](https://files.fm/f/aphm4uu7p)

**Comment by frenck** [comment]
Hi there @Marlo461 👋 

Your question is mostly very specific and related to Z-Wave JS or Z-Wave JS UI, not the add-on. You are probably better of seeking help @ Z-Wave JS.

Add-ons wrap existing software, in this case, Z-Wave JS UI. We don't modify or maintain the working of the application itself.

../Frenck

**Reply by Marlo461** [reply]
Thanks, I'll try
